testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -1108,11 +1108,26 @@ public void testCreateSchema()
         }
         assertThat(computeActual("SHOW SCHEMAS").getOnlyColumnAsSet()).doesNotContain(schemaName);
         assertUpdate("CREATE SCHEMA " + schemaName);
+
+        // verify listing of new schema
         assertThat(computeActual("SHOW SCHEMAS").getOnlyColumnAsSet()).contains(schemaName);
+
+        // verify SHOW CREATE SCHEMA works
         assertThat((String) computeScalar("SHOW CREATE SCHEMA " + schemaName))
                 .startsWith(format("CREATE SCHEMA %s.%s", getSession().getCatalog().orElseThrow(), schemaName));
+
+        // try to create duplicate schema
         assertQueryFails("CREATE SCHEMA " + schemaName, format("line 1:1: Schema '.*\\.%s' already exists", schemaName));
+
+        // use new schema
+        assertUpdate("CREATE TABLE " + schemaName + ".t(x int)");
+        assertQueryFails("DROP SCHEMA " + schemaName, ".*(Schema not empty|Cannot drop).*");
```

Comment:
I moved the assertion from Memory's connector test.
However this doesn't work for some of other connectors yet, eg MySQL.
hence https://github.com/trinodb/trino/discussions/8613 discussion

Reply:
created https://github.com/trinodb/trino/issues/8634 for follow up